orm/encoding/ormfield/bytes.go

```diff
@@ -87,10 +87,9 @@ func (b NonTerminalBytesCodec) Decode(r Reader) (protoreflect.Value, error) {
 func (b NonTerminalBytesCodec) Encode(value protoreflect.Value, w io.Writer) error {
 	bz := value.Bytes()
 	n := len(bz)
-	if n > 255 {
-		return ormerrors.BytesFieldTooLong
-	}
-	_, err := w.Write([]byte{byte(n)})
+	var prefix [binary.MaxVarintLen64]byte
```

Comment:
why allocating so much? We easily compute exactly how big the slice should be.

Reply:
unless there's a reason to believe this causes heap allocation (which it shouldn't), i'd prefer to leave as is because the current version uses less CPU

Reply:
yes, it doesn't matter. I added one more comment, but it seams I didn't send it. `MaxVarintLent64` is only 6.